greek/marshmallow.md

```diff
@@ -22,8 +22,4 @@
 
 [Προσφέρεις στο ζαχαρωτό τσιγάρο](offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md)
 
-[Πιάνεις το σχοινί](use_the_rope/use_the_rope.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.